### PR TITLE
Moved app to backend and renamed sql deployment to attempt to get rou…

### PR DESCRIPTION
**Pull request by mwevill**
…nd caching issue

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -14,10 +14,10 @@
         "serviceName": {
             "type": "string"
         },
-        "sharedFrontEndAppServicePlan": {
+        "SharedBackEndAppServicePlanName": {
```
Comment (dalemorson) [inline azure/template.json]: Change to camelCase to match other parameters 👍

File: azure/template.json [hunk 2]
```diff
@@ -153,7 +153,7 @@
                         "value": "[variables('appServiceName')]"
                     },
                     "appServicePlanName": {
-                        "value": "[parameters('sharedFrontEndAppServicePlan')]"
+                        "value": "[parameters('SharedBackEndAppServicePlanName')]"
```
Comment (dalemorson) [inline azure/template.json]: Same for this one, change to camelCase.

File: azure/template.json (hunk 1 above)
Comment (mwevill) [reply azure/template.json]: Fixed

File: azure/template.json (hunk 2 above)
Comment (mwevill) [reply azure/template.json]: Fixed

**Pull request merged**